esti/system_test.go

```diff
@@ -172,7 +176,7 @@ func uploadContent(ctx context.Context, repo string, branch string, objPath stri
 	if err != nil {
 		return nil, fmt.Errorf("write content: %w", err)
 	}
-	w.Close()
+	_ = w.Close()
```

Comment:
This Close can fail and destroy the test. Can we fail of this happens?

Reply:
Added check and to fail the upload in case of an error.